docs/source/contributing/documentation.rst

```diff
@@ -3,8 +3,80 @@ Adding Documentation
 ====================
 
 When submitting a new class through a PR, or any changes in general,
-there should be documentation where possible. Here are the guidelines
-for writing it.
+there should be documentation where possible. Here are our guidelines
+for writing documentation.
+
+Guidelines for examples
+-----------------------
+
+Everybody is welcome to contribute examples to the documentation. Since straightforward examples are a great resource for quickly learning manim, here are some guidelines.
+
+What makes a great example
+--------------------------
+
+.. note:: 
+
+   As soon as a new version of manim is released, the documentation will be a snapshot of that version. Examples contributed after the release will only be shown in the latest documentation.
+   
+* Examples should be ready to copy and paste for use.
+
+* Examples should be brief yet still easy to understand.
+
+* Examples don't require the ``from manim import *`` statement, this will be added automatically when the docs are build```
+
+* There should be a balance of animated and non-animated examples.
+
+- As manim makes animations, we can include lots of animated examples; however, our RTD has a maximum 15 minutes to build. Animated examples should only be used when necessary, as last frame examples render faster.
+
+- Lots of examples (e.g. size of a plot-axis, setting opacities, making texts, etc.) will also work as images. It is a lot more convenient to see the end product immediately instead of waiting for an animation to reveal it.
+
+* Examples should not have comments in the code. Explanations should be added outside of the example code.
+
+* Please ensure the examples run on the current master when you contribute an example.
+
+How examples are structured
+---------------------------
+
+* Examples can be organized into chapters and subchapters.
+
+- When you create examples, the beginning example chapter should focus on only one functionality. When the functionality is simple, multiple ideas can be illustrated under a single example.
+
+- As soon as simple functionalities are explained, the chapter may include more complex examples which build on the simpler ideas.
+
+Writing examples
+~~~~~~~~~~~~~~~~
+
+When you want to add/edit examples, they can be found in the ``docs/source/`` directory, or directly in the manim source code (e.g. ``manim/mobject/mobject.py``). The examples are written in ``rst`` format and use the manim directive (see :mod:`~.manim_directive` ), ``.. manim::``. Every example is in its own block, and looks like this:
+
+ .. code:: rst
```

Comment:
There is a leading space which causes the block below to be rendered within a block quote. Please fix.

```suggestion
 .. code:: rst
```

Reply:
Editing your suggestion as it doesn't remove the space (GitHub is weird and always adds an extra space when I copy/paste too btw, not sure why :/).